FactionGrinderClassic.lua

```diff
@@ -354,12 +354,12 @@ function FGC_DisplaySelectedCategories()
 
         win = _G["FGC_"..faction.."GrindStatsTrackerCheckbox"]
         win:SetPoint("TOPLEFT",70,top+3)
-        win:SetChecked(FGCSettings["GrindStatsTrackersShown"][faction])
+        win:SetChecked(FGCSettings["GrindStatsTrackersShown"][faction] == 1)
```

Comment:
It was broken because I am setting this to zero for false, and that was being detected as true...